makefile.example

```diff
@@ -49,7 +49,7 @@ up:
 # ==========================================================
 # Migrate the DB up by 1
 up-by-one:
-	bash -c "goose -dir=$(MIGRATION_DIR)"
+	bash -c "goose -dir=$(MIGRATION_DIR) up-by-one"
```

Comment:
added missing keyword `up-by-one`

Reply:
I kind-of understand `up-by-one` but we need more info on readme (it is like from 1.3 to 1.4, right?)

Reply:
up-by-one only run next migration, up runs all available migrations